### PR TITLE
feat: migration export script

### PR DESCRIPTION
**Pull request by ssvegaraju**
Issue #, if available:

Description of changes:
Added export functionality and helper scripts. Tested by exporting on test account.

Checklist:

- [X] Have you successfully deployed to an AWS account with your changes?
- [X] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: fwoa-tools/src/export-script.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    # Rename exported files into ndjson files
+    print('Renaming files')
+    client = boto3.client('s3')
+
+    response = client.list_objects(
+        Bucket=bucket_name,
+        Prefix=job_id,
+    )
+
+    regex_pattern = '\/partitionKeyDup=(\w+)(\w{8}-\w{4}-\w{4}-\w{4}-\w{12})\/run-\d{13}-part-r-(\d{5})'
```
Comment (karina-aws) [inline fwoa-tools/src/export-script.py:266]: Can you add beginning and end anchors here for exact pattern matching https://quip-amazon.com/9v4sA7lCuHSb/Regular-Expressions#temp:C:ZNYb552e89d3f9a484195d5c7ddc

Comment (ssvegaraju) [reply fwoa-tools/src/export-script.py]: Updated to add the end anchor. Unfortunately as this is part of an s3 uri, the pattern is only looking for the end of item key. The rest of the string would contain the s3 bucket and folder name as well, so we can't add the start anchor here. Verified that the export completes successfully with the end anchor though!

**Comment by kcadette** [comment]
Have you merged the latest develop branch to your branch?

**Comment by ssvegaraju** [comment]
> Have you merged the latest develop branch to your branch?

Not yet, these prs are being merged into a feature branch first.

**Pull request merged**